community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInDbmsProcedures.java

```diff
@@ -55,6 +70,20 @@
                 .map( FunctionResult::new );
     }
 
+    public static class ConfigResult
```

Comment:
It is public, but it's only ever used in this file (for the procedure). And I was only following the style previously set in this file (which is not necessarily ideal).